### PR TITLE
AutoMod

### PR DESCRIPTION
**Pull request by r888888888**
Start collecting data from approval/rejection events to see if a model can be trained to accurately predict post status.

### CODE REVIEW

**Comment by r888888888** [comment]
Started analyzing the data and got some interesting results. The script used to train the model using random forest: https://gist.github.com/r888888888/8937273341cbdfbbb615f667f38bfdc5

Training set of 750k samples, test set of 250k samples.

Feature importances:
```
[('copyright_identified', 0.00152194821184457),
 ('translated', 0.002549041710415309),
 ('artist_identified', 0.004330100088479885),
 ('note_count', 0.0056871543166959184),
 ('is_questionable', 0.008185238593698756),
 ('character_identified', 0.010585515428110316),
 ('comment_count', 0.011205108983305938),
 ('is_explicit', 0.01978964376104364),
 ('is_safe', 0.02698760873857485),
 ('uploader_median_score', 0.09139280087312251),
 ('tag_count', 0.10295850222671461),
 ('width', 0.10708183705178144),
 ('height', 0.10808656710676903),
 ('fav_count', 0.11414088738370166),
 ('file_size', 0.11918170261733999),
 ('artist_count', 0.1291240358709359),
 ('copyright_count', 0.1371923070374658)]
```

`artist_count` is the number of posts under the artist. `copyright_count` is similar for copyright.

Individually these don't look impressive, but aggregated:

```
              precision    recall  f1-score   support

         0.0       1.00      1.00      1.00     50076
         1.0       1.00      1.00      1.00    200924

   micro avg       1.00      1.00      1.00    251000
   macro avg       1.00      1.00      1.00    251000
weighted avg       1.00      1.00      1.00    251000
```

And this is before any serious hyperparameter tuning.
